5calls/app/src/main/java/org/a5calls/android/a5calls/model/DatabaseHelper.java

```diff
@@ -164,6 +180,21 @@ private void addContact(String contactId, String contactName) {
                 SQLiteDatabase.CONFLICT_IGNORE);
     }
 
+    public void addStarredIssue(String issueId) {
+        ContentValues values = new ContentValues();
+        values.put(StarredIssuesColumns.ID, issueId);
+        values.put(StarredIssuesColumns.TIMESTAMP, mTimeProvider.currentTimeMillis());
+        getWritableDatabase().insert(STARRED_ISSUES_TABLE_NAME, null, values);
+    }
+
+    public void removeStarredIssue(String issueId) {
+        getWritableDatabase().delete(
```

Comment:
please add a test that there's no issue if we remove an issue that's not in in the starred issues DB.